composer.json

```diff
@@ -16,5 +16,8 @@
         "psr-0": {
             "": "app/code/community"
         }
+    },
+    "extra": {
+        "magento-version-ee": "^1.12.0"
```

Comment:
it does not add version checker as a dependency, it tells to version checked that this module is compatible with said magento version.